lib/babel-config.js

```diff
@@ -22,7 +22,7 @@ function validate(conf) {
 		return {testOptions: {}};
 	}
 
-	if (!conf || typeof conf !== 'object' || !conf.testOptions || typeof conf.testOptions !== 'object' || Array.isArray(conf.testOptions) || Object.keys(conf).length > 1) {
+	if (!conf || typeof conf !== 'object' || !conf.testOptions || typeof conf.testOptions !== 'object' || Array.isArray(conf.testOptions)) {
```

Comment:
The `Object.keys()` bit was there to ensure only known keys were used. So we still need that, but it's gotten a bit trickier since more than 1 key is allowed.

It should be possible to provide `extensions` and not `testOptions`.